src/components/DateRangeFilter/DateRangeFilter.tsx

```diff
@@ -44,18 +44,11 @@ export const DateRangeFilter = ({ withRevokedRange, className }: DateRangeFilter
     navigate({
       to: '.',
       search: (old) => {
-        if ('createdRange' in old) {
-          delete old.createdRange;
-        }
-        if ('updatedRange' in old) {
-          delete old.updatedRange;
-        }
-        if ('revokedRange' in old) {
-          delete old.revokedRange;
-        }
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const { createdRange, updatedRange, revokedRange, ...other } = old;
```

Comment:
I suppose it didn't work because the keys were removed on original `old` object - so it had some "sideeffects".

In your new solution I don't like the need of disabling eslinter... 

Maybe try to copy the oryginal object 🤔 and then delete or use `map`.


Example:
```
const keysToRemove = ['createdRange', 'updatedRange', 'revokedRange'];
const newObject = { ...old };

keysToRemove.forEach(key => {
  delete newObject[key];
});
```